public/blog/github-trending-tab.md

```diff
@@ -0,0 +1,46 @@
+## News
+
+When I learned GitHub’s plan to remove the Trending tab. My immediate reaction was:
+
+![throw.gif](/blog/assets/github-trending-tab/throw.gif)
+
+For open source projects, showing up on the GitHub Trending Tab is a braggable milestone. As someone from the [HN](https://news.ycombinator.com/item?id=32681319) suggests, the Trending Tab sets on many developer’s bookmark bar:
+
+![hn.webp](/blog/assets/github-trending-tab/hn.webp)
```

Comment:
nit: To be consistent with the contents later, they should be `[throw]` and `[hn]`.

Reply:
Done